### PR TITLE
Use Slot gender mechanism for the Tinkerbots

### LINKED ISSUE #216: Tinkerbots slot gender use
The Tinkerbots model do not use the gender slot feature while it could be a good improvement.

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix #216

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: projects/robots/kinematics/tinkerbots/protos/TinkerbotsBrickAdapter.proto [hunk 1]
```diff
@@ -21,7 +21,7 @@ PROTO TinkerbotsBrickAdapter [
           rotation 0 1 0 1.57
           children [
             Slot {
-              type "lego cross female"
+              type "lego cross-"
               endPoint IS slot
```
Comment (stefaniapedrazzi) [inline projects/robots/kinematics/tinkerbots/protos/TinkerbotsBrickAdapter.proto]: Is it correct that here is `-`, what does it refer to?
I see two male plugs behind the block.

Note that `slot` is not defined in the PROTO header.

Comment (fabienrohrer) [reply projects/robots/kinematics/tinkerbots/protos/TinkerbotsBrickAdapter.proto]: You're right, this slot should be dropped.

**Comment by fabienrohrer** [comment]
> It would be good to use the gender slot also for the tinkerbots slots type.

Done.

> Then, we should also update the assets of the robot designer.

Yes, I will open an issue about this in the robot designer repo. As this is "internal stuff" (in Webots and in the robot designer assets), it doesn't affect the robot designer behavior, i.e., the robot designer export-to-webots feature works with or without this PR (I tested it). We can merge this PR safely.

**Comment by fabienrohrer** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**